docs/FeatureManagement/FeatureManagement.v1.0.0.schema.json

```diff
@@ -0,0 +1,31 @@
+{
+    "definitions": {},
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://azconfig.io/schemas/FeatureManagement/v1.0.0/FeatureManagement.json",
+    "type": "object",
+    "title": "An Azure App Configuration Feature Management Configuration",
+    "required": [
+        "FeatureManagement"
+    ],
+    "properties": {
+        "FeatureManagement":{
+            "$id": "#/properties/FeatureManagement",
+            "type": "object",
+            "title": "The Feature Management Schema",
+            "required": [
+                "FeatureFlags"
+            ],
+            "properties": {
+                "FeatureFlags": {
+                    "$id": "#/properties/FeatureManagement/properties/FeatureFlags",
+                    "type": "array",
+                    "title": "Feature Flags",
+                    "description": "Declares feature flags based on Microsoft Feature Flag schema.",
+                    "items": {
+                        "$ref": "http://azconfig.io/schemas/FeatureManagement/v1.1.0/FeatureFlag.json"
```

Comment:
$ref should be the uri of the file rather than the $id. I believe `./FeatureFlag.v1.1.0.schema.json` would work, but to make it work without folder context- I think we should do: `https://github.com/Azure/AppConfiguration/blob/main/docs/FeatureManagement/FeatureFlag.v1.1.0.schema.json`